### PR TITLE
Update player_input.rst

### PR DESCRIPTION
**Pull request by AstrayDev**
Changed the description of binding the "gamepads A button" for jumping to "joypads Button 0" which is more universal.

### CODE REVIEW

File: getting_started/first_3d_game/02.player_input.rst [hunk 1]
```diff
@@ -149,8 +149,7 @@ your interface should look like this.
 
 |image15|
 
-The final action to set up is the ``jump`` action. Bind the Space key and the gamepad's
-A button.
+The final action to set up is the ``jump`` action. Bind the Space key and joypad Button 0.
```
Comment (Calinou) [inline getting_started/first_3d_game/02.player_input.rst:152]:
```suggestion
The final action to set up is the ``jump`` action. Bind the Space key and
Joypad Button 0 (Bottom Action, Sony Cross, Xbox A, Nintendo B).
```

Comment (AThousandShips) [reply getting_started/first_3d_game/02.player_input.rst:152]: Indeed, `A` might be the internal name it is not universal and using it is misleading, providing a list of the proper names would be the best solution

Comment (AstrayDev) [reply getting_started/first_3d_game/02.player_input.rst]: Sorry, I could've been more descriptive. I made an update to list the buttons that button 0 is mapped to.

File: getting_started/first_3d_game/02.player_input.rst [hunk 2]
```diff
@@ -149,8 +149,7 @@ your interface should look like this.
 
 |image15|
 
-The final action to set up is the ``jump`` action. Bind the Space key and the gamepad's
-A button.
+The final action to set up is the ``jump`` action. Bind the Space key and joypad Button 0 (Bottom action, Sony Cross, Xbox A, Nintendo B).
```
Comment (AThousandShips) [inline getting_started/first_3d_game/02.player_input.rst:152]:
```suggestion
The final action to set up is the ``jump`` action. Bind the Space key and
Joypad Button 0 (Bottom Action, Sony Cross, Xbox A, Nintendo B).
```
Split up the long line, and capitalize more words, as per the original suggestion

**Review by AThousandShips — APPROVED** [review]
Please update your changes with `git commit --amend` to not make more commits, and please squash your commits into one, see [here](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase)

**Comment by AstrayDev** [comment]
I'll fix the description and commits when I get the chance, sorry for all the problems.

This is my first ever pr, so I mostly made it as a learning experience, thank you for your patience

**Pull request closed without merging**